### PR TITLE
Email template broken

### PR DESCRIPTION
**Pull request by mrvisser**
Ensure lines in email don't get too long

### CODE REVIEW

**Comment by mrvisser** [comment]
Assigning to @simong for review

File: node_modules/oae-activity/emailTemplates/mail.html.jst [hunk 1]
```diff
@@ -8,21 +8,21 @@
...
-        str += '            <tr id="greeting">';
-        str += '                <td><h1>' + greeting + '</h1></td>';
-        str += '            </tr>';
-        str += '            <tr id="summary" class="muted">';
-        str += '                <td class="wrapped">' + shared.getEmailSummary(util, user.emailPreference, activities, baseUrl) + '</td>';
-        str += '            </tr>';
-        str += '        </table>';
-        str += '    </td>';
-        str += '</tr>';
+        str += '<tr id="header" class="row">\n';
```
Comment (nicolaasmatthijs) [inline node_modules/oae-activity/emailTemplates/mail.html.jst]: Rather than manually appending `\n` everywhere, you could just push the lines in an array and join them at the end

File: node_modules/oae-activity/tests/test-email.js [hunk 2]
```diff
@@ -1474,7 +1474,7 @@ describe('Activity Email', function() {
             refreshConfiguration(null, false, false, {}, function(config) {
 
                 // Trigger an email with a long display name
-                var displayName = TestsUtil.generateRandomText(30);
+                var displayName = TestsUtil.generateRandomText(3);
```
Comment (nicolaasmatthijs) [inline node_modules/oae-activity/tests/test-email.js]: Why is this change necessary ?

File: node_modules/oae-email/lib/test/util.js [hunk 3]
```diff
@@ -102,6 +105,16 @@ var collectAndFetchAllEmails = module.exports.collectAndFetchAllEmails = functio
                     // Collect and send the emails
                     ActivityEmail.collectAllBuckets(function() {
                         EmailAPI.removeListener('debugSent', _handleDebugSent);
+
+                        // Ensure no email has an html content line longer than 997 chars (999 chars
+                        // minus CR+LF) to ensure we don't have any issues with postfix line-wrap
```
Comment (nicolaasmatthijs) [inline node_modules/oae-email/lib/test/util.js]: Can you put an `@see` to this ticket here?

**Comment by nicolaasmatthijs** [comment]
Finished review. Back to @mrvisser for follow-up

File: node_modules/oae-activity/tests/test-email.js (hunk 2 above)
Comment (mrvisser) [inline node_modules/oae-activity/tests/test-email.js]: Because 30 words of text makes for an excessively long line.

**Comment by mrvisser** [comment]
Fix for template issues in #1168

**Comment by mrvisser** [comment]
Followed up, back to @nicolaasmatthijs
